service/routes.py

```diff
@@ -173,6 +173,30 @@ def get(self, shopcart_id):
         app.logger.info("Returning shopcart: %s", shopcart.id)
         return shopcart.serialize(), status.HTTP_200_OK
 
+    @api.doc("update_shopcarts")
+    @api.response(404, "Shopcart not found")
+    @api.response(400, "The posted Shopcart data was not valid")
+    @api.expect(shopcart_base_model)  # Updated the expect decorator
```

Comment:
`@api.response(415, "Invalid header content-type")` is also required

Reply:
done